FirebaseInstallations/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # v7.0.0 -- M82
 - [changed] The global variable `FIRInstallationsVersionStr` is deleted.
   `FirebaseVersion.version()` or `[FIRVersion version]` should be used instead.
+- [changed] Throw an exception if `FIROptions.projectID` is missing or
```

Comment:
just to confirm:
`appId` is already checked by other Firebase processes, right?
Should we check it again or can we rely on Firebase Common to have this check?

Reply:
`appID` is checked by FirebaseCore SDK. Since Installations SDK is not meant to work without Core I don't think we should check it one more time.